util.js

```diff
@@ -48,4 +48,19 @@ exports.next = function (createStream, opts, property, range) {
 }
 
 
+exports.ScrollNotify = function (div, scroller) {
+  var cl = 'hypertabs--notify'
+  scroller.observ(function (len) {
+    if(div.classList.contains(cl) != (len != 0)) {
+      if(len) div.classList.add(cl)
```

Comment:
while you're at it you could do something like `div.dataset.unreadCount = len` so we can get a counter up there.